frontend/package.json

```diff
@@ -33,6 +33,7 @@
     "json-schema": "^0.3.0",
     "json2csv": "^5.0.6",
     "jsonwebtoken": "^8.5.1",
+    "moment": "^2.29.1",
```

Comment:
we have `dayjs` in our dependencies that should do everything `moment` does (since `moment` is getting deprecated) - will it be too much work to change it over? if it is we can just leave as is

Reply:
I tried replacing it but it made some date invalid, I'll keep moment for now and change to dayjs in the next ticket for adding person icon functionality